README.md

```diff
@@ -36,6 +36,7 @@ Examples of unacceptable behavior include:
   without their explicit permission
 * Other conduct which could reasonably be considered inappropriate in a
   professional setting
+* If there is a feature you want to be added, be patient, or be willing to put in the effort to make it possible. Do not demand that others do the work for you.
```

Comment:
Trying to make this wording fit better with the rest of the unacceptable behavior examples

```suggestion
*  Demading others do work for you. Instead, either ask nicely and be patient, or put in the effort to make it possible. 
```

Reply:
Also, we should probably put this above the "other" line, that feels like it makes sense as the last thing on the list

Reply:
Fair point. Updated.